src/theme.js

```diff
@@ -29,22 +37,60 @@ export const grommetToolsTheme = {
            "Apple Color Emoji", 
            "Segoe UI Emoji", 
            "Segoe UI Symbol"`,
+    },
+  },
+  anchor: {
+    color: 'black',
+  },
+  box: {
+    size: {
+      headerSize: '36px',
+      xlarge: '200px',
+    },
+  },
+  textInput: {
+    input: {
+      font: {
+        size: '18px',
+      },
+    },
+  },
+  formField: {
+    border: {
+      position: null,
```

Comment:
Are you using the FormField component?

Reply:
Right now FormField is being used for the search bar. I think we could just use a text input by itself. On the designs it isn't clear if there is going to be a search bar there or a drop down so Joe and I haven't messed with it too much

Reply:
Search shouldn't be using FormField, we can remove this entry from the theme.